hole/css-colors-inverse.go

```diff
@@ -0,0 +1,27 @@
+package hole
+
+var cssFixedTests []test
+var cssCodesToNames = make(map[string][]string)
+
+func initCSS() {
+	if len(cssCodesToNames) < 1 {
+		cssFixedTests = fixedTests("css-colors")
```

Comment:
This doesn't need to be a package global, it's only used in this func, use `:=`, or better still just fold it directly into the range.